src/test/java/com/es/phoneshop/dao/CustomOrderDaoTest.java

```diff
@@ -6,21 +6,23 @@
 import org.junit.Test;
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
 
 import java.util.ArrayList;
 import java.util.Optional;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 public class CustomOrderDaoTest {
 
     @Mock
     private Order order;
+    @Spy
```

Comment:
what is the reason of spy here?

Reply:
in placeOrder test i verify that save method of dao was called